### ISSUE TITLE
Tensorboard integration

### ISSUE THREAD

**Issue by SaiKrishnaBala** [mistralai/mistral-finetune#32]
What configuration changes needs to be done in yaml so that it emits loss values which cab be fed to tensorboard.
I have tried all the steps and i'm able to monitor the training loss values on wandb portal but i would like to have a local tensorbaord setup for this.
Any references would be helpful.

**Comment by juliendenize** [comment]
Hi! We're archiving this repository as it's no longer actively maintained, and therefore this issue is being closed.
